frontend/class-opengraph-image.php

```diff
@@ -563,15 +563,9 @@ protected function is_valid_image_url( $url ) {
 	 *
 	 * @return string The path of the image URL. Returns an empty string if URL parsing fails.
 	 */
-	protected function get_image_url_path( $url ) {
-		$parsed_url = wp_parse_url( $url );
-
-		if ( $parsed_url === false ) {
-			return '';
-		}
-
-		return $parsed_url['path'];
-	}
+	 protected function get_image_url_path( $url ) {
```

Comment:
```
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 562 | ERROR | [x] Line indented incorrectly; expected 1 tabs, found
     |       |     1 (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
 564 | ERROR | [x] Line indented incorrectly; expected 1 tabs, found
     |       |     1 (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
```
Please remove 1 space before protected function (leave only 1 tab) and same before closure of function. (It should be into line with `/*`).

Reply:
Sorry for that i fix that. I have Atom with editorconfig.